src/interface/stemplot.jl

```diff
@@ -0,0 +1,133 @@
+"""
+`stemplot(v; nargs...)`->` Plot`
+
+Description
+===========
+
+Draws a stem leaf plot of the given vector 'v'.
+
+Usage
+=====
+
+`stemplot(v)`
+
+function stemplot(
+                  v::Vector,
+                  symbol::AbstractString, 
+                  scale::Int64)
+Arguments
+=========
+
+-**`v`** : Vector for which the stem leaf plot should be computed
+-**`symbol`: Symbol for break between stem and leaf. default = "|"
+-**`expand`**: Bool for expanded (true) plot or collapsed (false). default = true
+-**`color`**: Set color of plot. default = white TODO
+
+Results
+=======
+
+A plot of object type 
+
+TODO
+
+Author(s)
+========
+- Iain Dunning (Github: https://github.com/IainNZ)
+- Christof Stocker (Github: https://github.com/Evizero)
+- Kenta Sato (Github: https://github.com/bicycle1885)
+- Alex Hallam (Github: https://github.com/alexhallam)
```

Comment:
Did you reuse code from other places of the package? if not than you should be the only one listed here